### PR TITLE
Minor code improvements

### PR DESCRIPTION
**Pull request by eexit**
- [x] Use single quotes when possible
- [x] Improve readability of string constructs
- [ ] ~~Flags `Graphite::push()` as deprecated (shouldn't be part of the public interface)~~

### CODE REVIEW

File: src/Beberlei/Metrics/Collector/Graphite.php [hunk 1]
```diff
@@ -110,4 +115,22 @@ public function push($stat, $value, $time = null)
...
+
+    /**
+     * @deprecated
+     * @todo Makes this method private, move pushStat() logic into this one afterward
+     *
+     * @param $stat
+     * @param $value
+     * @param int|null $time
+     */
+    public function push($stat, $value, $time = null)
```
Comment (lyrixx) [inline src/Beberlei/Metrics/Collector/Graphite.php]: As said, I **want** to keep this method in the public API.

File: src/Beberlei/Metrics/Collector/Librato.php [hunk 2]
```diff
@@ -77,9 +77,11 @@ public function flush()
             return;
         }
 
+        $authToken = base64_encode($this->username.':'.$this->password);
+
         try {
             $this->browser->post('https://metrics-api.librato.com/v1/metrics', array(
-                'Authorization: Basic '.base64_encode($this->username.":".$this->password),
+                'Authorization: Basic '.$authToken,
```
Comment (lyrixx) [inline src/Beberlei/Metrics/Collector/Librato.php]: This change should be reverted because it allocated a new vars that you use only once.

Comment (eexit) [inline src/Beberlei/Metrics/Collector/Librato.php]: as you wish

**Comment by eexit** [comment]
PR updated.

**Comment by lyrixx** [comment]
Thank you Joris.

**Comment by eexit** [comment]
@lyrixx Could you make a new release with the changes please?

Thanks!

**Comment by lyrixx** [comment]
@eexit sure ;)

**Comment by lyrixx** [comment]
@eexit here we go: https://github.com/beberlei/metrics/releases/tag/v2.0.2

**Comment by eexit** [comment]
@lyrixx Thanks man! :)

**Pull request merged**